apps/src/blockly/customBlocks/googleBlockly/mutators/behaviorGetMutator.js

```diff
@@ -12,7 +13,11 @@ export const behaviorGetMutator = {
   },
 
   // Only used to save in XML, but still required to exist by Blockly.
-  mutationToDom: function () {},
+  mutationToDom: function () {
```

Comment:
Note: This function still isn't called since we don't save in XML, however it does what we'd expect it to do if call manually. This felt better than leaving a stub.

Reply:
Updated!